frontend/src/metabase/home/containers/HomepageApp.jsx

```diff
@@ -4,14 +4,11 @@ import { connect } from "react-redux";
 import { t } from "ttag";
 import { push } from "react-router-redux";
 
-import Greeting from "metabase/lib/greeting";
```

Comment:
This shouldn't be removed. It's what powers the "MetaBot" greetings in the top-left of the homepage:

<img width="238" alt="image" src="https://user-images.githubusercontent.com/2223916/102655334-fe8abd00-4126-11eb-9269-cbddadf9c6ec.png">

Reply:
I still have it in `localhost` on the homepage, after having removed this.
![image](https://user-images.githubusercontent.com/31325167/102655707-0d975c80-4173-11eb-8edb-b1a6fde16422.png)